lib/ui/viewer/file/zoomable_image.dart

```diff
@@ -112,21 +100,19 @@ class _ZoomableImageState extends State<ZoomableImage>
       content = const EnteLoadingWidget();
     }
 
-    final GestureDragUpdateCallback? verticalDragCallback = _isZooming
-        ? null
-        : (d) => {
-              if (!_isZooming)
+    verticalDragCallback(d) => {
+          if (_photoViewController.scale! <= _initialScale!)
```

Comment:
Can you not compare this with `PhotoViewScaleState.initial` like how it was being done previously?

Reply:
for that, we need _scaleStateChangedCallback, which then breaks the sliding